Helper/Api.php

```diff
@@ -148,6 +148,12 @@ public function sendApiRequest($sRequestUrl)
 
         $aResponse = $this->formatOutputByResponse($aResponse);
 
+        if (!array_key_exists('status', $aResponse)) {
+            $aResponse['status'] = 'ERROR';
+            $aResponse['errorcode'] = '903';
```

Comment:
I would argue that one shouldn't map this to a (theoretically) possible error code from the Payone platform to avoid confusion. Maybe use errorcode '0' with an appropriate message?